src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -214,13 +231,13 @@ public function handle()
             "has_frames_tabs"   => $this->getApplicationPaths(),
 
             "tables"      => [
-                "GET Data"              => $_GET,
-                "POST Data"             => $_POST,
-                "Files"                 => $_FILES,
-                "Cookies"               => $_COOKIE,
-                "Session"               => isset($_SESSION) ? $_SESSION :  [],
-                "Server/Request Data"   => $_SERVER,
-                "Environment Variables" => $_ENV,
+                "GET Data"              => $this->maskBlacklisted($_GET, $this->blacklist['_GET']),
+                "POST Data"             => $this->maskBlacklisted($_POST, $this->blacklist['_POST']),
+                "Files"                 => $this->maskBlacklisted($_FILES, $this->blacklist['_FILES']),
+                "Cookies"               => $this->maskBlacklisted($_COOKIE, $this->blacklist['_COOKIE']),
+                "Session"               => isset($_SESSION) ? $this->maskBlacklisted($_SESSION, $this->blacklist['_SESSION']) :  [],
+                "Server/Request Data"   => $this->maskBlacklisted($_SERVER, $this->blacklist['_SERVER']),
+                "Environment Variables" => $this->maskBlacklisted($_ENV, $this->blacklist['_ENV']),
```

Comment:
There’s a bit too much noise for the simple change in functionality. I would like to ask you to improve this a bit.

The simplest change is to not repeat `this->blacklist`, since `maskBlacklisted` can get it itself: `$this->maskBlacklisted($_GET, '_GET')`. Next, we can rename the function to be simpler, since it’s a private function: `$this->masked($_GET, '_GET')`. Finally, we can remove the duplication if we rely on `$GLOBALS`, which is not a big deal here: `$this->masked('_GET')`, and `$GLOBALS[$name]` in the function.

Feel free to do your own changes, as long as the end result is considerably less noisy than what we see now.